app/api/setup.py

```
@@ -0,0 +1,63 @@
+            os.path.dirname(os.path.abspath(settings.credentials_file)), exist_ok=True
+        )
+
+        # Save to settings.credentials_file
+        with open(settings.credentials_file, "wb") as f:
+            f.write(content)
+
+        logger.info(f"Credentials uploaded successfully to {settings.credentials_file}")
+        return {
+            "message": "Credentials uploaded successfully",
+            "path": settings.credentials_file,
+        }
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)